enviro/provisioning.py

```diff
@@ -207,4 +207,4 @@ def catchall(request):
 logging.info("  - client connected!", ap.status("stations")[0])
 
 logging.info("> running provisioning application...")
-server.run(host="0.0.0.0", port=80)
\ No newline at end of file
+server.run(host="0.0.0.0", port=80)
```

Comment:
Very minor thing, but this file is coming up as changed because there was a new line added at the end. I suggest undoing it just so it makes this PR cleaner

Reply:
sounds good, I'll revert this change.